ffi/src/lib.rs

```diff
@@ -1245,9 +1246,55 @@ pub unsafe extern "C" fn tx_report_entity_for_temp_id(tx_report: *mut TxReport,
     }
 }
 
-// TODO: cache
+/// Adds an attribute to the cache.
+/// `store_cache_attribute_forward` caches values for an attribute keyed by entity
+/// (i.e. find values and entities that have this attribute, or find values of attribute for an entity)
+///
+/// # Safety
```

Comment:
My comments on the earlier PR, about the relatively low value of these particular safety blocks, are relevant here.